openadapt/events.py

```diff
@@ -484,24 +526,23 @@ def is_same_pos(e0, e1):
                 return False
         return True
 
-
-    def should_discard(event, prev_event, next_event):
-        return (
-            event.name == "move" and (
-                is_same_pos(prev_event, event) or
-                is_same_pos(event, next_event)
-            )
+    def should_discard(event: Any, prev_event: Any, next_event: Any) -> bool:
```

Comment:
`event: ActionEvent, prev_event: ActionEvent | None, next_event: ActionEvent | None`

Reply:
Addressed in https://github.com/OpenAdaptAI/OpenAdapt/pull/289/commits/d88767ff785961d7eb36ca589d96f707a019b507